server/middleware/populateSelectedEstablishment.ts

```diff
@@ -0,0 +1,14 @@
+import type { RequestHandler } from 'express'
+import asyncMiddleware from './asyncMiddleware'
+
+export default function populateSelectedEstablishment(): RequestHandler {
+  return asyncMiddleware(async (req, res, next) => {
+    if (res.locals.feComponentsMeta != null) {
+      const { activeCaseLoad } = res.locals.feComponentsMeta
+      req.session.prisonId = activeCaseLoad.caseLoadId
+      req.session.prisonName = activeCaseLoad.description
```

Comment:
Aha. This is getting the prisonId and name populated - from frontend-components. So I'll change the ticket to use this value, or stub it in tests.